js/main.js

```diff
@@ -0,0 +1,27 @@
+const returnRandomIntegers = (from, to) => {
```

Comment:
Существует такой прием, как "ранний выход из функции", это когда мы с помощью условий обрабатываем сначала негативные сценарии, а после обрабатываем положительный. Это позволяет коду выглядеть более чисто, т.к. у нас будут очень маленькие блоки с условиями и это помогает избежать вложенности условий.
Например
```suggestion
const getRandomIntegers = (from, to) => {
  if (from < 0 || to < 0) {
    throw new Error('Диапазон должен включать только положительные числа');
  }

  if (from === to) {
    throw new Error('Диапазон не может состоять из равных значений');
  }

  if (from > to) {
      to = from;
      from = to;
  }

  return Math.floor(Math.random() * (to - from + 1) ) + from;
}
```

Попробуй переписать обе функции таким образом.